packages/filecmd.rb

```diff
@@ -38,7 +29,12 @@ def self.check
   end
   
   def self.install
-    system "install -Dm755 filefix #{CREW_DEST_PREFIX}/bin/filefix"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install-strip"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.install 'filefix', "#{CREW_DEST_PREFIX}/bin/filefix", mode: 0755
+    system "make DESTDIR=#{CREW_DEST_DIR} install-strip | sed -e \"s,libtool: warning: remember to run 'libtool --finish #{CREW_LIB_PREFIX}',,g\""
```

Comment:
Why do we need to remove the warning?

Reply:
We already done it in `postinstall` part

Reply:
I think it would be better if we kept the warning message.  What does it hurt?